### PR TITLE
Fix segmentation test

### PR DESCRIPTION
**Pull request by KuangHaofei**
The `model.hybridize` in scripts/segmentation/test.py file cause the file cannot run. Comment this line could fixed the problem.

### CODE REVIEW

File: scripts/segmentation/test.py [hunk 1]
```diff
@@ -241,15 +241,15 @@ def benchmarking(model, args):
             else:
                 raise RuntimeError("=> no checkpoint found at '{}'" \
                     .format(args.resume))
-        if args.quantized:
-            model.hybridize(static_alloc=True, static_shape=True)
-        else:
-            model.hybridize()
+        # if args.quantized:
```
Comment (bryanyzhu) [inline scripts/segmentation/test.py]: Please don't comment these two lines. This may break quantization code.

File: scripts/segmentation/test.py [hunk 2]
```diff
@@ -241,15 +241,15 @@ def benchmarking(model, args):
...
+        #    model.hybridize(static_alloc=True, static_shape=True)
+        # else:
```
Comment (bryanyzhu) [inline scripts/segmentation/test.py]: You can simply delete these two lines. We only comment explanations for code.

**Comment by mli** [comment]
Job PR-1078-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1078/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1078-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1078/2/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/segmentation/test.py [hunk 3]
```diff
@@ -242,9 +242,7 @@ def benchmarking(model, args):
                 raise RuntimeError("=> no checkpoint found at '{}'" \
                     .format(args.resume))
         if args.quantized:
-            model.hybridize(static_alloc=True, static_shape=True)
-        else:
-            model.hybridize()
+           model.hybridize(static_alloc=True, static_shape=True)
```
Comment (hetong007) [inline scripts/segmentation/test.py]: need one more space for indentation.

**Comment by mli** [comment]
Job PR-1078-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1078/3/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/segmentation/test.py [hunk 4]
```diff
@@ -88,7 +88,7 @@ def parse_args():
     args = parser.parse_args()
     
     args.ctx = [mx.cpu(0)]
-    args.ctx = [mx.gpu(i) for i in range(args.ngpus)] if args.ngpus > 0 else args.ctx
+    args.ctx = [mx.gpu(i+4) for i in range(args.ngpus)] if args.ngpus > 0 else args.ctx
```
Comment (hetong007) [inline scripts/segmentation/test.py]: what's this `+4` for?

Comment (KuangHaofei) [reply scripts/segmentation/test.py]: While I test the code, my first 4 gpu are used in training another model. So I use `+4` to make the program running in last 4 gpu. I am so sorry that I don't modify it before I push the code.

Comment (hetong007) [reply scripts/segmentation/test.py]: Whenever open a PR, please always double check the diff, especially for such a short one. If there's any mistake, just fix it and push.

**Comment by mli** [comment]
Job PR-1078-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1078/4/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**